index.android.d.ts

```diff
@@ -276,7 +276,18 @@ declare module 'react-native-google-fit' {
     bucketUnit: BucketUnit
   };
 
-  type rawSteps = Array<{startDate: string, endDate: string, steps: number}>;
+  type rawSteps = Array<{
+    dataTypeName: string,
+    dataSourceId: string,
+    originDataSourceId: string,
+    deviceUid?: string,
+    deviceManufacturer?: string,
+    deviceModel?: string,
+    deviceType?: string,
```

Comment:
I guess that this could also be typed to match the exact strings that we return.

Reply:
look good to me